subsystem/cluster_test.go

```diff
@@ -139,5 +142,18 @@ var _ = Describe("system-test cluster install", func() {
 		for _, host := range c.GetPayload().Hosts {
 			Expect(swag.StringValue(host.Status)).Should(Equal("installing"))
 		}
+
+		file, err := ioutil.TempFile("", "tmp")
+		if err != nil {
```

Comment:
maybe should be
```suggestion
		Expect(err).NotTo(HaveOccurred())
```
You can fix the other place that doing the same as well.